plugins/modules/win_zip.ps1

```diff
@@ -49,11 +52,11 @@ try {
 Function Compress-Zip($src, $dest) {
     If (-not $module.CheckMode) {
         If (Test-Path -LiteralPath $src -PathType Container) {
-            [System.IO.Compression.ZipFile]::CreateFromDirectory($src, $dest, $compressionLevel, $srcWildcard)
+            [System.IO.Compression.ZipFile]::CreateFromDirectory($src, $dest, $compressionLevel, (-not $srcWildcard), $encoding)
         } Else {
             $zip = [System.IO.Compression.ZipFile]::Open($dest, 'Update')
             try {
-                [System.IO.Compression.ZipFileExtensions]::CreateEntryFromFile($zip, $src, $srcFile, $compressionLevel)
+                [void][System.IO.Compression.ZipFileExtensions]::CreateEntryFromFile($zip, $src, $srcFile, $compressionLevel)
```

Comment:
`CreateEntryFromFile` outputs data, to avoid it clashing with the module return json and showing an error we just void it out.